### ISSUE TITLE
Disable `D401` for `pytest.fixture`

### ISSUE THREAD

**Issue by NMertsch** [astral-sh/ruff#22890]
### Summary

Pytest's [fixtures](https://docs.pytest.org/en/stable/explanation/fixtures.html) are functions, but used as values. I think [D401 - non-imperative-mood](https://docs.astral.sh/ruff/rules/non-imperative-mood/) should not apply to these fixtures.

```python
import pytest

@pytest.fixture
def one() -> int:
    """The number ``1``."""  # <- D401: First line of docstring should be in imperative mood
    return 1
```

Why is this relevant? IDEs like PyCharm display fixture docstrings on usage:
```python
def test_add(one: int) -> None:
    assert one + one == 2  # hover on `one` shows fixture docstring
```

[Playground](https://play.ruff.rs/36e88e2b-5ef6-41b8-84cf-7202c1b425de)

### Version

ruff 0.14.14 (8b2e7b36f 2026-01-22)

**Comment by ntBre** [comment]
This makes some sense to me, but I think it may be a good use for the [per-file-ignores](https://docs.astral.sh/ruff/settings/#lint_per-file-ignores) setting to ignore `D401` for your test directory. Alternatively, you could add `pytest.fixture` to the [ignore-decorators](https://docs.astral.sh/ruff/settings/#lint_pydocstyle_ignore-decorators) or [property-decorators](https://docs.astral.sh/ruff/settings/#lint_pydocstyle_property-decorators) options.

**Comment by NMertsch** [comment]
> This makes some sense to me, but I think it may be a good use for the [per-file-ignores](https://docs.astral.sh/ruff/settings/#lint_per-file-ignores) setting to ignore `D401` for your test directory. Alternatively, you could add `pytest.fixture` to the [ignore-decorators](https://docs.astral.sh/ruff/settings/#lint_pydocstyle_ignore-decorators) or [property-decorators](https://docs.astral.sh/ruff/settings/#lint_pydocstyle_property-decorators) options.

Thank you. Yes, these options work, but:
* per-file-ignores disables D401 for all functions in the files, not just fixtures.
* ignore-decorators disables all D rules, not just D401.
* property-decorators probably does what I want, but fixtures are not properties, so it feels hacky.

I think the default behavior could be improved here, but I also know it is a very minor and niche suggestion. So if the ruff team thinks this doesn't make enough sense or isn't worth the effort of implantation and maintenance, I'm perfectly fine with the status quo :)

**Comment by ntBre** [comment]
Yep, all the workarounds have downsides :) Let's keep this open and see if others are interested in it too!

**Comment by AH-Hammi** [comment]
I would also appreciate such a feature, as this could really help improving writing and reading pytests with fixtures.
